content/dstu2/openid-connect.md

```diff
@@ -0,0 +1,88 @@
+---
+title: OpenID Connect | DSTU 2 API
+---
+# Authorization Server OpenID Connect Support #
+---------------------------------------------
+###Introduction###
+
+[OpenID Connect][OPENID] describes itself as "a simple identity layer 
+on top of the OAuth 2.0 protocol".  The specification provides a set
+of message structures, a messaging protocol, and a security framework
+to allow a system that has authenticated a user to securely convey said
+identity to another service provider (relying party).  Cerner's 
+authorization server implements support for OpenID Connect in
+conjunction with the [SMART on FHIR<sup>�</sup>][SMART] profile 
+of OAuth 2.  Client applications may use this as a source of 
+"single sign-on" for users when protecting resources outside of the EMR 
+(example: user-generated data that is stored by the client application).
+
+In addition to SSO, SMART on FHIR<sup>�</sup> also leverages OpenID 
+Connect to convey the location of the authenticated user's 
+FHIR<sup>�</sup> endpoint, as described by the HL7<sup>�</sup>
```

Comment:
FHIR endpoint may be confused with the Endpoint resource. This is the profile url as specified by SMART, correct? Which happens to be an implementation of a FHIR resource?